templates/provider/hmc/templates/rbac/user-facing/capi-identities-editor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "hmc.fullname" . }}-capi-identities-editor-role
+rules:
+  - apiGroups:
+      - infrastructure.cluster.x-k8s.io
+    resources:
+      - awsclusterroleidentities
+      - awsclusterstaticidentities
+      - azureclusteridentities
+      - vsphereclusteridentities
```

Comment:
`awsclustercontrolleridentities` should be also added to have all CAPA cluster identity types

Reply:
`awsclustercontrolleridentities` doesn't make sense for us as we don't have any default credentials used by capa controllers